### PR TITLE
fix: arrows in note review panel don't move

### PR DESCRIPTION
**Pull request by 4Source**
I had a look at this bug #934 and implemented a fix. Just a few changed where required. 
Mostly I changed the classes the nav items should have to match the structure of obsidian file hierarchy. 

- [x] Test run successfully 
Test Suites: 19 passed, 19 total
Tests:       206 passed, 206 total
- [x] Fixes Issue

### CODE REVIEW

File: src/gui/Sidebar.tsx [hunk 1]
```diff
@@ -173,18 +171,8 @@ export class ReviewQueueListView extends ItemView {
         }
 
         folderTitleEl.onClickEvent(() => {
-            for (const child of childrenEl.childNodes as NodeListOf<HTMLElement>) {
-                if (child.style.display === "block" || child.style.display === "") {
-                    child.style.display = "none";
-                    (collapseIconEl.childNodes[0] as HTMLElement).style.transform =
-                        "rotate(-90deg)";
-                    deck.activeFolders.delete(folderTitle);
```
Comment (4Source) [inline src/gui/Sidebar.tsx:181]: What is this for?

Comment (4Source) [reply src/gui/Sidebar.tsx:181]: this line: deck.activeFolders.delete(folderTitle);

Comment (ronzulu) [reply src/gui/Sidebar.tsx:181]: This stores the collapsed/expanded state of each folder (folder names being things like
    // "TODAY", "NEW" or formatted dates).

So here where the tree node is being collapsed, the folder name is being removed

Comment (4Source) [reply src/gui/Sidebar.tsx:181]: Is the call for this function at this spot correct it seems for me not necessary?

Comment (ronzulu) [reply src/gui/Sidebar.tsx:181]: Like me, do you also think that the `activeFolders` is used to maintain the state of which folders are expanded?
This code is being executed when the user clicks on an expanded folder name, causing it to collapse.
So it seems to me that calling `deck.activeFolders.delete(folderTitle);` is valid.

Comment (4Source) [reply src/gui/Sidebar.tsx:181]: Yeah I think your right I will fix this

**Review by st3v3nmw — APPROVED** [review]
LGTM

**Pull request merged**